pkg/vectorize/bin/bin.go

```diff
@@ -87,3 +88,24 @@ func uintToBinary(x uint64) string {
 
 	return string(b[i+1:])
 }
+
+
+func Bin[T constraints.Unsigned | constraints.Signed](xs []T, rs []string) []string {
+	for idx := range xs {
```

Comment:
i don't known what will happen when some item is null.
just add some unit test to check

Reply:
ok